region/maridia/inner-pink/West Cactus Alley Room.json

```
@@ -302,6 +302,88 @@
         "With a second-to-last-frame jump it is still possible but more precise."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Side Platform Cross Room Jump",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 2.4375,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "note": [
+                "This applies to Seaweed Room, Skree Boost Room, Grapple Tutorial Room 2, Warehouse Energy Tank Room, and Warehouse Entrance."
+              ],
+              "devNote": [
+                "With Speed Booster and a Spring Ball jump, this can also work with shorter runways, as in Ice Beam Snake Room and Grapple Beam Room,",
+                "but it wouldn't be very useful, because with those items there are alternative strats using the doorway runway."
+              ]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 4,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "note": [
+                "This applies to Ridley Tank Room and Dust Torizo Room."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 7.4375,
+              "speedBooster": false,
+              "obstructions": [[1, 0]],
+              "requires": [
+                "HiJump"
+              ],
+              "note": [
+                "This applies to Big Boy Room and Mickey Mouse Room."
+              ],
+              "devNote": [
+                "With a momentum-conserving turnaround, this could be done with shorter runway,",
+                "but there aren't rooms in the game that it would apply to."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 7.4375,
+              "speedBooster": false,
+              "obstructions": [[3, 2]],
+              "requires": [
+                "HiJump",
+                "canMomentumConservingMorph",
+                "canInsaneJump",
+                {"or": [
+                  "h_heatProof",
+                  "canInsaneMidAirMorph"
+                ]}
```

```suggestion
                "canMomentumConservingMorph"
```
Easier than most of these strats.  And you can farm cacs to make up for heat damage.  Hitbox the pirates or let them move on their own.